components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/TestConstants.java

```diff
@@ -0,0 +1,28 @@
+package org.wso2.carbon.identity.oauth2;
+
```

Comment:
IMO would be better to have these constants declared in the test class itself unless you plan to reuse them in other tests.

Reply:
Some of them will be used it seems. (very common things). Hence it's fine IMO :)

Reply:
Instead are u sure all these constants are used ?

Reply:
I removed unused constants. But there are two similar classes such as Saml1BearerGrantHandler  and Saml2BearerGrantHandler. they may use those constants.